content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -20,6 +20,46 @@ To access the Scans Beta you must be an org owner or org member. Staff org membe
 
 We use the following IP Address to make requests to your target: **52.19.40.38**
 
+## What is a DAST Target?
+
+A DAST target is the specific entry point (URL or endpoint) of a web application, website, API, or any component that accepts input from the outside world. It defines the scope, or boundaries, of the security scan conducted by a DAST tool, limiting the tool to analyzing only those pages, links, or forms within the target's domain. 
+
+For instance, with a target of `https://example.com`, the scan would cover `https://example.com/app1` but not `https://app2.example.com`. Essentially, the scanner examines URLs beginning with "example.com."
+
+### Examples of DAST Targets:
+
+- **URLs:** These are the most common targets, representing individual web pages or sections within an application:
+
+  - `https://www.example.com`: Top-level domain hosting the application
+
+  - `https://example.com/blog`: Sub-folder or sub-section within the application
+
+  - `https://admin.example.com`: Subdomain hosting a separate application
+
+- **APIs:** Applications often use APIs for communication, and DAST can identify vulnerabilities like unauthorized access or data leaks:
+
+  - `https://api.us.example.com`: Internal API hosted on a subdomain
+
+  - `https://api.eu.example.com`: Separate API instance hosted independently
+
+- **Forms:** DAST can assess the security of login forms, registration pages, and other input fields against attacks like SQL injection or cross-site scripting:
```

Comment:
In essence, the form URL is not different from subfolders in the first section like `https://example.com/blog`? Does it have to be highlighted in a separate section?

Reply:
@eugenerevzin what are your thoughts?

Reply:
@mtsuji-cobalt - Would it be difficult to make this it's own page, under the Scans section?  The content looks great, but marketing team wanted to see if we could do it standalone for easier linking.

Reply:
@eugenerevzin we can create a new page but this section also does have a direct link via `platform-deep-dive/scans/#what-is-a-dast-target`